stdlib/testing/testdata/conv_bool_to_float.flux

```diff
@@ -0,0 +1,33 @@
+package testdata_test
+
+import "testing"
+
+option now = () => (2030-01-01T00:00:00Z)
+
+inData = "
+#datatype,string,long,dateTime:RFC3339,boolean,string,string
+#group,false,false,false,false,true,true
+#default,_result,,,,,
+,result,table,_time,_value,_field,_measurement
+,,0,2018-05-22T19:53:30Z,true,k,m
+,,0,2018-05-22T19:53:31Z,false,k,m
+"
+
+// NOTE: This test will fail with differences in the last two rows when time zone support arrives.
```

Comment:
What do you mean?

Reply:
This was a relic of copy&paste from  original totime.flux i took as an example. I totally overlooked it, sorry.

Reply:
👍  No worries